tests/BaselineSearch/test_baseline_search.py

```diff
@@ -24,7 +24,7 @@ def run_test_get_unprocessed_stack_params(product):
 
     original_properties = product['properties']
     assert(original_properties['polarization'] in params.polarization)
-    assert(['VV', 'VV+VH'] == params.polarization)
+    assert(['VV', 'VV+VH'] == params.polarization if reference.properties['polarization'] in ['VV', 'VV+VH'] else ['HH','HH+HV'] == params.polarization)
```

Comment:
Be careful with bursts! Burst can't be dual-pol, so a VV+VH SLC will yield a bunch of VV bursts and a bunch of VH bursts, which can only be stacked with like-pol bursts. Will probably need some `if()` here to constrain it to the ref burst pol only if it's a burst stack.

Reply:
Made it so burst product baseline params will only use the burst product's polarization